tex/chapters/08-Conclusion.tex

```diff
@@ -0,0 +1,51 @@
+%!TEX root = ../main.tex
+
+\chapter{Conclusion}\label{cha:conclusion}
+
+The main aim of this report was to assess the current tools for distinguishing between strategies in the Iterated Prisoner's Dilemma and implement them within the Axelrod-Python library.
+The report also aimed to produce a novel approach to the problem and use this to perform some analysis on the strategies contained within Axelrod-Python.
+Ultimately both of these goals were achieved and some preliminary analysis of the Axelrod-Python was completed.
+In particular, an \textbf{original} results is presented in Section \ref{sec:fsm_proof} allowing the work completed in previous literature to be extended further.
```

Comment:
`result`

also add another sentence highlighting the original work of the previous two chapters: no one has done that before, it's good stuff!